server/events.go

```diff
@@ -415,11 +426,16 @@ func (s *Server) sendShutdownEvent() {
 	s.mu.Unlock()
 	// Send to the internal queue and mark as last.
 	si := &ServerInfo{}
-	sendq <- &pubMsg{nil, subj, _EMPTY_, si, si, noCompression, true}
+	sendq <- &pubMsg{nil, subj, _EMPTY_, si, nil, si, noCompression, false, true}
```

Comment:
If fields would be named, only the calls where we actually pass the non default values would need to be changed, but I understand that this was done this way so that adding a new field would fail compilation, forcing you to review all placed where pubMsg is initialized.